Makefile

```diff
@@ -8,8 +8,9 @@ GO_BIN := ${GOPATH}/bin
 LINT_BIN := $(GO_BIN)/golangci-lint
 GOACC_BIN := $(GO_BIN)/go-acc
 
-LINT_COMMIT := v1.46.0
-GOACC_COMMIT := 80342ae2e0fcf265e99e76bcc4efd022c7c3811b
+LINT_VERSION := v1.60.1
```

Comment:
If we're at it, could we also use the Dockerized linter that we use everywhere else, so we don't have to have multiple versions installed?